examples/material_gallery/lib/demo/weathers_demo.dart

```diff
@@ -0,0 +1,495 @@
+// Copyright 2015 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:async';
+import 'dart:ui' as ui;
+import 'package:flutter/material.dart';
+import 'package:flutter/services.dart';
+import 'package:flutter_sprites/flutter_sprites.dart';
+
+ImageMap _images;
```

Comment:
Agreed on the use of globals are not ideal. However, I'm going to leave it for now as there seems to currently not be a super good way of loading assets. Ideally, I'd like to see something like how you reference the built in icons (where they can be accessed from anywhere in the code without passing around lots of references). Will file an issue if there isn't one already.